node_modules/oae-principals/lib/internal/dao.js

```diff
@@ -81,9 +81,9 @@ var createUser = module.exports.createUser = function(user, callback) {
  * @param  {Group}      callback.group      The created group
  */
 var createGroup = module.exports.createGroup = function(groupId, tenantAlias, displayName, description, visibility, joinable, createdBy, callback) {
-    var lastModified = Date.now();
+    var lastModified = new Date();
     var query = 'INSERT INTO "Principals" ("principalId", "tenantAlias", "displayName", "description", "visibility", "joinable", "lastModified", "createdBy", "created") VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)';
-    var parameters = [groupId, tenantAlias, displayName, description, visibility, joinable, lastModified.toString(), createdBy, lastModified.toString()];
+    var parameters = [groupId, tenantAlias, displayName, description, visibility, joinable, '' + lastModified.getTime(), createdBy, lastModified];
```

Comment:
Is this equivalent to `lastModified.getTime().toString()`? I'd prefer that if possible